protostar/commands/test/test_command.py

```diff
@@ -108,6 +108,15 @@ def arguments(self) -> List[Command.Argument]:
                 type="int",
                 description="Set a seed to use for all fuzz tests.",
             ),
+            Command.Argument(
+                name="max-fuzz-examples",
```

Comment:
Wouldn't it be more consistent if all fuzzing-related options had single prefix `--fuzz-*` ?